spec/models/task_spec.rb

```diff
@@ -1,5 +1,37 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
+  describe 'validation' do
+    it 'is valid with all attributes' do
+      task = build(:task)
+      expect(task).to be_valid
+      expect(task.errors).to be_empty
+    end
+
+    it 'is invalid without title' do
+      task_without_title = build(:task, title: "")
```

Comment:
下のテストケースが`nil`を指定しているので、記法を合わせても良いかもしれません。
```suggestion
      task_without_title = build(:task, title: nil)
```